src/components/previewers/ImagePreviewer.jsx

```
@@ -0,0 +1,150 @@
+
+const ImagePreviewer = ({
+  image,
+  isOpen,
+  onClose,
+  isTop = false,
+  likes = 0,
+  comments = [],
+  commentText,
+  setCommentText,
+  totalComments = 0,
+  OnLike,
```

ESLint

[error] 'OnLike' is missing in props validation

(react/prop-types)